regression/contracts/assigns_replace_09/test.desc

```diff
@@ -0,0 +1,12 @@
+CORE
+main.c
+--replace-call-with-contract bar
+^EXIT=0$
+^SIGNAL=0$
+\[\d+\] file main.c line \d+ Check that bar\'s assigns clause is a subset of foo\'s assigns clause: SUCCESS$
+^VERIFICATION SUCCESSFUL$
+--
+^Condition: \!not\_found$
+--
+Checks whether CBMC properly evaluates subset relationship on assigns
```

Comment:
We already have a regression test for subset checking. May be clarify here that this is specifically for `static` functions whose names are mangled?

Reply:
Implemented as suggested.